test/test_player_info.py

```diff
@@ -0,0 +1,11 @@
+from testing_base import *
+
+def PlayerInfo():
+    player=Player("Jacob", 200)
+    player_info = ""
```

Comment:
Can't you just call your function from the source by using Player.info(player) and then assert that equals your output?

Reply:
The way I was testing was making sure that the info was in the in the string. If so, it would be true. If not for some reason, then it would be false.